_observing-your-data/metricsanalytics.md

```diff
@@ -0,0 +1,205 @@
+
+---
+
+## Configuring Prometheus to send metrics data to OpenSearch
+
+You must first create a connection from [Prometheus](https://prometheus.io/) to OpenSearch using the [SQL plugin](https://github.com/opensearch-project/sql). You can then configure a connection to Prometheus by using the `_datasources` configuration API endpoint. 
+
+The following example shows a request that sets up a Prometheus data source without any authentication:
+
+```json
+POST _plugins/_query/_datasources 
+{
+    "name" : "my_prometheus",
+    "connector": "prometheus",
+    "properties" : {
+        "prometheus.uri" : "http://localhost:9090"
+    }
+}
+```
+{% include copy-curl.html %}
+
+The following example shows how to set up a Prometheus data source using AWS Signature Version 4 authentication:
+
+```json
+POST _plugins/_query/_datasources
+{
+    "name" : "my_prometheus",
+    "connector": "prometheus",
+    "properties" : {
+        "prometheus.uri" : "http://localhost:8080",
+        "prometheus.auth.type" : "awssigv4",
+        "prometheus.auth.region" : "us-east-1",
+        "prometheus.auth.access_key" : "{{accessKey}}"
+        "prometheus.auth.secret_key" : "{{secretKey}}"
+    }
+}
+```
+{% include copy-curl.html %}
+
+After setting up the Prometheus to OpenSearch connection, you can view Prometheus metrics in OpenSearch Dashboards by going to the **Observability** > **Metrics analytics** page, where the metrics will be displayed, similar to the following image.
+
+<img src="{{site.url}}{{site.baseurl}}/images/metrics/metrics1.png" alt="Prometheus metrics displayed on a dashboard" width="700"/>
+
+### Developer resources
+
+See the following developer resources for sample code, articles, tutorials, and API references:
+
+* [Datasource Settings](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/admin/datasources.rst), which contains information about authentication and authorization of data source APIs.
+* [Prometheus Connector](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/admin/connectors/prometheus_connector.rst), which contains configuration information.
+* [Simple Schema for Observability](https://github.com/opensearch-project/opensearch-catalog/tree/main/docs/schema/observability), which contains information about the OTel schema and ingest pipeline.
+* [OTel Metrics Source](https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-plugins/otel-metrics-source), which contains information about the Data Prepper metrics pipeline and ingestion.
+
+---
+
+## Experimenting with OTel Metrics in the OpenSearch demo environment 
+
+The OpenSearch [`opentelemetry-demo` repository](https://github.com/opensearch-project/opentelemetry-demo) is a practical demonstration of collecting, processing, and visualizing metrics data through **OTel Metrics** from OpenTelemetry, following OpenTelemetry standards, and using the **Metrics analytics** tool in OpenSearch Dashboards.
```

Comment:
Is the "following OpenTelemetry standards" phrase necessary, or is this implied? Also, "OpenTelemetry" => "OTel".